Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
```

Comment:
Yes, I saw this in DSCResource.tools too. And I think a few others. Totally agree with you - needs a few more opinions.